packages/mdx/test/compile.js

```diff
@@ -858,13 +858,11 @@ test('jsx', async () => {
     [
       '/*@jsxRuntime automatic @jsxImportSource react*/',
       'function _createMdxContent(props) {',
-      '  const _components = Object.assign({',
-      '    "a-b": "a-b"',
-      '  }, props.components);',
-      '  return <>{<_components.a-b></_components.a-b>}</>;',
+      '  const _components = props.components || ({}), _component0 = "a-b";',
+      '  return <>{<_component0></_component0>}</>;',
       '}',
       'function MDXContent(props = {}) {',
-      '  const {wrapper: MDXLayout} = props.components || ({});',
+      '  const _component0 = "a-b", {wrapper: MDXLayout} = props.components || ({});',
```

Comment:
Same here as above?

Reply:
Ah, I think I understand this prop a bit better now. Looks like I didn't implement this right! I'll rework 👍

Reply:
Sweet, thanks!
You could also likely set it on that destructuring object, alongside `wrapper: MDXLayout`? You may need to set a computed field in the AST or so though 🤔: https://astexplorer.net/#/gist/702f0315bf76ddbaf45a0928b0b06e96/784539f882835483b4541f9eff8d81ec4a327733